extensions/pkg/predicate/predicate_test.go

```diff
@@ -87,6 +89,72 @@ var _ = Describe("Predicate", func() {
 		})
 	})
 
+	Describe("#AddTypePredicate", func() {
+		var (
+			object       client.Object
+			createEvent  event.CreateEvent
+			updateEvent  event.UpdateEvent
+			deleteEvent  event.DeleteEvent
+			genericEvent event.GenericEvent
+		)
+
+		BeforeEach(func() {
+			object = &extensionsv1alpha1.Extension{
+				Spec: extensionsv1alpha1.ExtensionSpec{
+					DefaultSpec: extensionsv1alpha1.DefaultSpec{
+						Type: extensionType,
+					},
+				},
+			}
+			createEvent = event.CreateEvent{
+				Object: object,
+			}
+			updateEvent = event.UpdateEvent{
+				ObjectOld: object,
+				ObjectNew: object,
+			}
+			deleteEvent = event.DeleteEvent{
+				Object: object,
+			}
+			genericEvent = event.GenericEvent{
+				Object: object,
+			}
+		})
+
+		It("should return true because only one extention exists and extention type matches", func() {
```

Comment:
```suggestion
		It("should return true because only one extension exists and extension type matches", func() {
```
Please correct the spelling throughout.